### PR TITLE
READMEbutbetter.md

### PR DESCRIPTION
**Pull request by Annahile**
horse moment
but better

### CODE REVIEW

**Comment by Annahile** [comment]
PLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASEPLEASE

**Comment by Annahile** [comment]
I'll let you see a real live horse instead of just one online :)))))))))

**Comment by nanowuu** [comment]
Prepare for unforeseen consequences

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ### cube simulator 3d 2
-it's cubin' time
\ No newline at end of file
+it's cubin' time
+print("I am a horse annd welcome to my ashow!!!")
```
Comment (nanowuu) [inline README.md:3]: this is utter garbage

**Review by nanowuu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nanowuu) [reply README.md:3]: improper language

**Review by nanowuu — CHANGES_REQUESTED** [review]
review

**Comment by nanowuu** [comment]
> review 

shut up

**Pull request closed without merging**